src/common/structures/IState.ts

```diff
@@ -0,0 +1,5 @@
+export default interface IState {
+	addingItem: boolean;
+	initialData: any;
+	unsavedData: any;
+}
```

Comment:
I'm not sure what this is for but it doesn't appear to be "common". I'd like to see this removed if it's not used for anything and moved to the specific component that uses it if that's the case.